include/mbgl/map/map.hpp

```diff
@@ -182,10 +183,11 @@ class Map : private util::noncopyable {
     double getDefaultPitch() const;
 
     // Feature queries
-    std::vector<Feature> queryRenderedFeatures(const ScreenCoordinate&, const QueryOptions& options = {});
-    std::vector<Feature> queryRenderedFeatures(const ScreenBox&,        const QueryOptions& options = {});
-    
+    std::vector<Feature> queryRenderedFeatures(const ScreenCoordinate&, const RenderedQueryOptions& options = {});
+    std::vector<Feature> queryRenderedFeatures(const ScreenBox&,        const RenderedQueryOptions& options = {});
+
     AnnotationIDs queryPointAnnotations(const ScreenBox&);
+    std::vector<Feature> querySourceFeatures(const std::string&,        const SourceQueryOptions& options = {});
```

Comment:
@jfirebaugh In this case; shall I move [SourceQueryOptions](https://github.com/mapbox/mapbox-gl-native/blob/03969287248730779ed65287f28c45f7cb566f04/include/mbgl/map/query.hpp#L22) to a separate header in the `mbgl::style` namespace? or keep it here?